assets/js/pages/SapSystemsOverviewPage/SapSystemsOverview.test.jsx

```diff
@@ -80,13 +90,69 @@ describe('SapSystemsOverviews component', () => {
         tenant
       );
       expect(mainRow.querySelector('td:nth-child(5)')).toHaveTextContent(
-        dbAddress
+        sapSystemType
       );
       expect(mainRow.querySelector('td:nth-child(6)')).toHaveTextContent(
+        dbAddress
+      );
+      expect(mainRow.querySelector('td:nth-child(7)')).toHaveTextContent(
         'ENSA1'
       );
     });
 
+    it('should display the correct SAP system type', () => {
+      const sapSystemTypes = [
+        'ABAP',
+        'J2EE',
+        'ABAP|J2EE',
+        'J2EE|ABAP',
+        'SOME_SAP_SYSTEM_FEATURE|NOT_A_REAL_SYSTEM',
+      ];
+
+      const expectedSapSystemTypes = [
+        'ABAP',
+        'JAVA',
+        'ABAP/JAVA',
+        'ABAP/JAVA',
+        '',
+      ];
+
+      const sapSystems = sapSystemFactory.buildList(5);
```

Comment:
Creating a factory to modify just after that looks like an anti-pattern. 
Why don't you create the value using the factory with the wanted entries?
Something maybe like:
```
const sapSystems = expectedSapSystemTypes
.map((type) => sapSystemFactory.build({ application_instances : applicationInstanceFactory.buildList(2, { features: type }))
```